includes/kp-functions.php

```diff
@@ -48,7 +48,7 @@ function get_klarna_customer( $customer_type ) {
 		$customer['klarna_access_token'] = $access_token;
 	}
 
-	return $customer;
+	return apply_filters( 'kp_set_customer_type', $customer );
```

Comment:
Pass the `$customer_type` variable to the filter as the 2nd parameter so people can see what we used to get the customer value initially.